pkg/portal/middleware/aad.go

```diff
@@ -157,8 +153,8 @@ func (a *aad) AAD(h http.Handler) http.Handler {
 			return
 		}
 
-		expires, ok := session.Values[SessionKeyExpires].(time.Time)
-		if !ok || expires.Before(a.now()) {
+		expires, ok := session.Values[SessionKeyExpires].(int64)
+		if !ok || time.Unix(expires, 0).Before(a.now()) {
 			h.ServeHTTP(w, r)
 			return
 		}
```

Comment:
@m1kola if someone sends an old value, we are treating it as not logged in and we will redirect them to /api/login from what I can see